src/components/core/pivotTable/aggregationsMenuHelper.ts

```diff
@@ -8,21 +8,26 @@ import { AVAILABLE_TOTALS } from '../../visualizations/table/totals/utils';
 import { IColumnTotal } from './AggregationsMenu';
 
 const getTotalTypesAppliedOnAllMeasures = (
-    columnTotals: AFM.ITotalItem[],
+    totals: AFM.ITotalItem[],
     measureLocalIdentifiers: string[]
-): AFM.TotalType[] => AVAILABLE_TOTALS.filter((type) => {
-    const columnTotalsLength = columnTotals.filter((total: AFM.ITotalItem) => total.type === type).length;
-    return columnTotalsLength === measureLocalIdentifiers.length;
-});
+): AFM.TotalType[] => {
+    return AVAILABLE_TOTALS.filter((type) => {
+        const totalsFiltered = totals.filter((total: AFM.ITotalItem) => total.type === type);
+        const measuresFiltered = totalsFiltered.map((total: AFM.ITotalItem) => total.measureIdentifier);
```

Comment:
I would rename the variable to `totalsMeasureIdentifiers`.

Reply:
fixed